src/bots/depositor.py

```diff
@@ -278,42 +277,22 @@ def _send_deposit_tx(
                 payload,
                 guardian_signs,
             )
-            success = self._send_transaction(deposit_tx)
+            success = self._send_transaction(deposit_tx, False)
 
         logger.info({'msg': f'Tx send. Result is {success}.'})
 
         self._flashbots_works = not self._flashbots_works or success
         return success
 
-    def _is_mellow_depositable(
-        self,
-        module_id: int
-    ) -> bool:
-        if not variables.MELLOW_CONTRACT_ADDRESS:
-            return False
-        staking_module_contract: StakingModuleContract = self.w3.lido.simple_dvt_staking_strategy.staking_module_contract
-        if staking_module_contract.get_staking_module_id() != module_id:
-            logger.debug(
-                {
-                    'msg': 'While building mellow transaction module check failed.',
-                    'contract_module': staking_module_contract.get_staking_module_id(),
-                    'tx_module': module_id
-                }
-            )
-            return False
-        vault_address = self.w3.lido.simple_dvt_staking_strategy.vault()
-        balance = staking_module_contract.weth_contract.balance_of(vault_address)
-        MELLOW_VAULT_BALANCE.labels(module_id).set(balance)
-        if balance < variables.VAULT_DIRECT_DEPOSIT_THRESHOLD:
-            logger.info({'msg': f'{balance} is less than VAULT_DIRECT_DEPOSIT_THRESHOLD while building mellow transaction.'})
-            return False
-        return True
-
     def _send_transaction(
         self,
-        tx: ContractFunction
+        tx: ContractFunction,
+        is_mellow: bool
```

Comment:
Looks useless here. Maybe just log in _send_deposit_tx method

Reply:
Done